### PR TITLE
Create Smart Router template

### PR DESCRIPTION
**Pull request by jakubschwan**

### CODE REVIEW

File: bpmsuite/bpmsuite70-smartrouter.json [hunk 1]
```diff
@@ -0,0 +1,298 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "description": "Application template for Red Hat JBoss BPM Suite 7.0 which includes a Business Central Monitoring and Smart Router.",
```
Comment (sutaakar) [inline bpmsuite/bpmsuite70-smartrouter.json]: Please update description - remove Business central part.

File: bpmsuite/bpmsuite70-smartrouter.json [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+            "iconClass": "icon-jboss",
+            "tags": "bpmsuite,jboss,xpaas",
+            "version": "1.4.0",
+            "openshift.io/display-name": "Red Hat JBoss BPM Suite 7.0 Business Central Monitoring + Smart router"
```
Comment (sutaakar) [inline bpmsuite/bpmsuite70-smartrouter.json]: Please update - remove Business central part.

File: bpmsuite/bpmsuite70-smartrouter.json [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+        },
+        "name": "bpmsuite70-businesscentral-monitoring-with-smartrouter"
```
Comment (sutaakar) [inline bpmsuite/bpmsuite70-smartrouter.json]: Please update

File: bpmsuite/bpmsuite70-smartrouter.json [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+    },
+    "labels": {
+        "template": "bpmsuite70-businesscentral-monitoring-with-smartrouter",
```
Comment (sutaakar) [inline bpmsuite/bpmsuite70-smartrouter.json]: Please update

File: bpmsuite/bpmsuite70-smartrouter.json [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+        "xpaas": "1.4.0"
+    },
+    "message": "A new BPM Suite Business Central Monitoring + Smart Router applications have been created in your project. The username/password for accessing the BPM Business Central Monitoring interface is \"${KIE_ADMIN_USER}/${KIE_ADMIN_PWD}\".",
```
Comment (sutaakar) [inline bpmsuite/bpmsuite70-smartrouter.json]: Please update message

**Review by sutaakar — COMMENTED** [review]
Looks good, just please update description - remove Business central mentions.

**Comment by jakubschwan** [comment]
@sutaakar thanks, good catch. I knew I forgot something.

**Review by sutaakar — APPROVED** [review]

**Pull request closed without merging**